### ISSUE TITLE
NTR: Autopsy: melanin reduced in substantia nigra (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9202]
New term request Autopsy: melanin reduced in substantia nigra (elthat_101115013149): Loss of pigmented neurons in the substantia nigra  UMLS: 1834856

**Comment by pnrobinson** [comment]
@MickeySegal

I would add the following term, OK?

Label: Loss of pigmented neurons in the substantia nigra
Def: Neuron loss in the substantia nigra, with a consequent reduction in neuromelanin concentration in the substantia nigra.
PMID:11724917

**Comment by MickeySegal** [comment]
That is good.

**Comment by pnrobinson** [comment]
Looking back at this I think this is essentially a synonym for 
[Substantia nigra dopaminergic neuron loss HP:0034694](https://hpo.jax.org/app/browse/term/HP:0034694).

See - [Parkinson's disease](https://www.sciencedirect.com/topics/medicine-and-dentistry/parkinsons-disease) results from loss of [dopaminergic neurons](https://www.sciencedirect.com/topics/medicine-and-dentistry/dopaminergic-neuron) in the [substantia nigra](https://www.sciencedirect.com/topics/medicine-and-dentistry/substantia-nigra) and locus caeruleus. These neurons normally contain [melanin](https://www.sciencedirect.com/topics/medicine-and-dentistry/melanin) pigment (neuromelanin), and loss of these neurons results in pallor of the substantia nigra that can be appreciated grossly in cross-sections of the midbrain.

I am thinking it is best to expand the definition of the term and to add "melanin reduced in substantia nigra" as a related synonym. Does this work?

**Comment by MickeySegal** [comment]
That is even better.

**Comment by pnrobinson** [comment]
OK, I updated the existing term with a comment and added the above as a related synonym.
